vizro-core/examples/_chart-gallery/pages/_factories.py

```diff
@@ -74,3 +74,39 @@ def butterfly_factory(group: str):
             make_code_clipboard_from_py_file("butterfly.py"),
         ],
     )
+
+
+def connected_scatter_factory(group: str):
+    """Reusable function to create the page content for the column chart with a unique ID."""
+    return vm.Page(
+        id=f"{group}-connected-scatter",
+        path=f"{group}/connected-scatter",
+        title="Connected Scatter",
+        layout=vm.Layout(grid=PAGE_GRID),
+        components=[
+            vm.Card(
+                text="""
+                #### What is a connected scatter chart?
```

Comment:
```
                #### What is a connected scatter chart?
```
Is there a reason you're starting with H4, rather than H1?

Reply:
It's inside a text card, so from a hierarchy view it's lower than the dashboard and page title (which are both H2). H3 still looked too big, so I opted for H4 😄

Reply:
I thought it must be something like that ... all good from my perspective.